ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java

```diff
@@ -33,9 +32,8 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
-    public void EndVisitTest() throws Exception {
+    public void EndVisitTest() {
```

Comment:
lets track this test again ,dropping the exception shouldn't be the reason why this test was failing

Reply:
> dropping the exception shouldn't be the reason why this test was failing,look more closely on how the method works ie EndVisitTest()

Did you take a look at this [commit](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/301/commits/bdfea9cf519a72bd9d95fc2c7f1007350bc5e4aa).
I think if this [issue](https://issues.openmrs.org/browse/RA-1895) is resolved then this node will be ready